azure-pipelines-v3.yml

```diff
@@ -176,17 +176,17 @@ jobs:
       DocsRoot:
         params: https://github.com/MicrosoftDocs/DocsRoot --timeout 20
       azure-docs-pr:
-        params: https://github.com/MicrosoftDocs/azure-docs-pr --timeout 90 --regression-markdown-rule --regression-metadata-schema
+        params: https://github.com/MicrosoftDocs/azure-docs-pr --timeout 90 --regression-markdown-rule --regression-metadata-schema --regression-allowlist
```

Comment:
Consider combine these `--regresion-xyz` into a single cmd args like `--regression-rules` (they tend to appear all at the same time).

Reply:
This PR will cover it https://github.com/dotnet/docfx/pull/6713, and @fkpwolf will add another --regression parameter.